### PR TITLE
clean: use native JS methods instead of JQuery

### PR DESCRIPTION
**Pull request by shenlebantongying**
I know a tiny bit of jQuery years ago, and I am not motivated to learn it more. Because learning standardized JS can be useful elsewhere but learning jQuery is more or less useless to me.

The `$_$` 🤑 is weird. Just use `jQuery`, the real symbol.

The prettier doesn't work for me locally, I adjusted it a little.

### CODE REVIEW

File: src/article_maker.cc [hunk 1]
```diff
@@ -49,8 +49,10 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word, QString const &
 
   // add jquery
   {
+    // TODO: document why Jquery cannot be removed or can we?
     result += R"(<script src="qrc:///scripts/jquery-3.6.0.slim.min.js"></script>)";
-    result += R"(<script> var $_$=jQuery.noConflict(); </script>)";
+    // TODO: document why noCliflict need to be called.
+    result += R"(<script> jQuery.noConflict(); </script>)";
```
Comment (shenlebantongying) [inline src/article_maker.cc:55]: I have vague memory that including jQuery actually fixes some dictionaries that forget to include jQuery.

However, I cannot find the source of this claim and the `jQuery.noConflict()` seems to nullify this claim.

We need to document this fact here if jQuery cannot be removed. @xiaoyifang

File: src/article_maker.cc [hunk 2]
```diff
@@ -63,14 +65,19 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word, QString const &
...
+         window.articleview = channel.objects.articleview;});
+   console.log("Qt webchannel ready...");
+  };
+
+  if (document.readyState !== "loading") {
+    gd_init_QtWebChannel();
+  } else {
+    document.addEventListener("DOMContentLoaded", gd_init_QtWebChannel);
+  };
+</script>
```
Comment (shenlebantongying) [inline src/article_maker.cc:80]: I think we should move these from `.cc` to `.js`. I will probably do it in another PR.

File: src/article_maker.cc (hunk 1 above)
Comment (xiaoyifang) [reply src/article_maker.cc:55]: I remembered that some dictionaries will include their own version of JQuery which will conflict with the built-in JQuery.

**Comment by xiaoyifang** [comment]
> The `$_$` 🤑 is weird

I agree ,it is just a trick to make sure that no others will use the same variable name.

File: src/article_maker.cc (hunk 1 above)
Comment (shenlebantongying) [reply src/article_maker.cc:55]: I think we have to keep it forever if it actually fixes something in the wild. Or someone implicitly have their own script invokes this jQuery.

I will just add comment about this reality.

**Comment by shenlebantongying** [comment]
There is another `jQuery`, I will remove it another day.

**Pull request merged**